prompts/demo_flow.ipynb

```diff
@@ -20,6 +20,7 @@
    ]
```

Comment:
<p>can delete all the cell output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/edu/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>